test/e2e/node/container_probe.go

```diff
@@ -33,36 +31,23 @@ import (
 
 const (
 	probTestContainerName       = "test-webserver"
-	probTestInitialDelaySeconds = 30
+	probTestInitialDelaySeconds = 15
```

Comment:
@timstclair I see, sorry for the trouble, :P.

FYI, we have a PR to make the test run parallel https://github.com/kubernetes/kubernetes/pull/29092.